1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 18 Line 1_
```markdown
### Inspecting commands
```
These sections are a nice list of the key bash commands!